.gitignore

```diff
@@ -4,3 +4,1150 @@ tools
 build
 *.o
 *.lst
+compiler-build/config.status
```

Comment:
A czy tutaj nie będzie pasować `compiler-build/*`?

Reply:
nawet wydaje mi się, że to jakiś nieaktualny wpis bo katalog "compiler-build" nie powinien sie tworzyc. Usuwam